### PR TITLE
fix(docker-compose-for-local): added mmi env in client-node

### PR DESCRIPTION
**Pull request by sahil-ondc**

### CODE REVIEW

File: docker-compose-for-local.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ services:
   nginx:
     build:
       context: biap-app-ui-front
-      dockerfile: Dockerfile
+      dockerfile: DockerfileWithoutSSL
```
Comment (navdeep710) [inline docker-compose-for-local.yaml:32]: is dockerfilewithoutssl updated

Comment (navdeep710) [reply docker-compose-for-local.yaml:32]: in here 
https://github.com/ONDC-Official/ondc-sdk/blob/d4bea5b6f37405956173c48f899ce20e637ef717/deploying_ansible/buyer-app-run.yaml#L74C5-L81C1

we are using normal dockerfile without ssl in ansible

Comment (sahil-ondc) [reply docker-compose-for-local.yaml:32]: understood, this was throwing error on local-setup, for now reverting this Dockerfile change, will fix with ansible together

**Comment by sahil-ondc** [comment]
@navdeep710 Please merge this MMI fix

**Pull request closed without merging**